ipinfo/main.go

```diff
@@ -57,6 +57,8 @@ func main() {
 		err = cmdCIDR2Range()
 	case cmd == "range2cidr":
 		err = cmdRange2CIDR()
+	case cmd == "range2ip":
+		err = cmdRange2IP()
```

Comment:
So we've added `range2ip` into the main IPinfo command, but we should also create a separate binary entirely called `range2ip`, the same way it's done for `range2cidr`, `cidr2range` and `grepip`. See those root folders to understand how it's done.